proposals/libocispec.md

```diff
@@ -0,0 +1,35 @@
+# OCI libocispec project proposal
+
+## Abstract
+Provide a C library for parsing and handle OCI runtime spec file and OCI image spec files.
+
+## Proposal
+With repositories under the http://github.com/opencontainers organization:
+
+Rename (transfer) https://github.com/giuseppe/libocispec would become https://github.com/opencontainers/libocispec.
+
+### Initial Maintainers
```

Comment:
submitted a new version with  Qiang Huang <h.huangqiang@huawei.com> who offered to be a maintainer as well.